include/transmem/transmem.h

```diff
@@ -59,41 +60,64 @@ class EmptyLinkQueryException : public std::runtime_error {
     Link emptyLink;
 };
 
-
-/****************************
- * PATH                     *
- ****************************/
+/********
+ * PATH *
+ ********/
 
 struct Path {
+
     FrameID src;
     FrameID dst;
     std::vector< std::reference_wrapper<Link> > links;
 
     void writeJSON(QJsonObject &json) const;
 };
 
+/************************************
+ * STAMPED AND RATED TRANSFORAMTION *
+ * **********************************/
+
+/* REMARK:
+   New return type of transmem. Encodes not just the transformation but also
```

Comment:
I inserted this remarks mainly for you to explain / clarify some ideas behind the implementation. They aren't intended to stay in the final version. Unfortunately I forget to tell you. :/